src/index.js

```diff
@@ -31,19 +31,44 @@ const closeArgsToError = (code, signal) => {
   return null
 }
 
+const concatBuffer = (buffer) => {
+  if (!(buffer instanceof Array)) {
+    // throw new Error('Expected buffer to be an Array')
+    return
```

Comment:
These cases most likely shouldn't occur but I still wanted to handle them, though I wasn't sure whether to throw errors or just return. Thoughts?